developer-kit/developer-kit.ino

```diff
@@ -1,6 +1,8 @@
 // This is an example project.
 
+#if defined(ARDUINO_ARCH_AVR)
 #include <avr/wdt.h>
+#endif
```

Comment:
Shouldnt this also remove watchdog timer stuff if its not supported?

Reply:
That include is AVR specific, and I was building on SAMD based board.

There actually no calls to any of the watch dog timer functions anywhere in this code, so the include can be removed altogether.